edenia/eden-spend-explorer
# Fix/treasury balance over time graph 368

### treasury balance over time graph

### What does this PR do?

- Resolve #368
- Resolve #386 

### Steps to test

1. Run Project
2. Enter to Income, must see someting like this:
![imagen](https://user-images.githubusercontent.com/49013556/211114662-143c1704-25c0-48eb-ba70-e39308ca7912.png)



#### CheckList
